internal/worker/uniter/remotestate/mock_test.go

```diff
@@ -290,22 +290,10 @@ func (u *mockUnit) WatchRelations() (watcher.StringsWatcher, error) {
 	return u.relationsWatcher, nil
 }
 
-func (u *mockUnit) WatchUpgradeSeriesNotifications(context.Context) (watcher.NotifyWatcher, error) {
```

Comment:
More to remove in this test file:

* ./internal/worker/uniter/remotestate/mock_test.go:229:	upgradeSeriesWatcher             *mockNotifyWatcher